sdks/python/apache_beam/runners/direct/util.py

```diff
@@ -47,6 +48,12 @@ def __init__(self, applied_ptransform, uncommitted_output_bundles,
     # Populated by the TransformExecutor.
     self.logical_metric_updates = None
 
+    self.step_context = (
```

Comment:
<!--new_thread; commit:6ce5f121f4998fe51aa9ac13064dce692cd1228b; resolved:0-->
Let's avoid making this a property for now (i.e., just keep this as a local variable) since we don't use it elsewhere.